nxdrive/client/local/linux.py

```diff
@@ -55,6 +56,9 @@ def set_folder_icon(self, ref: Path, icon: Path) -> None:
         """Create a special file to customize the folder icon."""
         log.debug(f"Setting the folder icon of {ref!r} using {icon!r}")
         # To be implementation with https://jira.nuxeo.com/browse/NXDRIVE-1831
+        subprocess.check_output(
```

Comment:
Could you add this code inside a try/except block to prevent crashes.In case of error, log a warning.

Do you know if that will work when there is a space or `'` in those paths?

Reply:
Also, have a look at the [macOS version](https://github.com/nuxeo/nuxeo-drive/blob/9102456ca8ce541439eb1948d0846e2967564c41/nxdrive/client/local/darwin.py#L66) and [Windows version](https://github.com/nuxeo/nuxeo-drive/blob/9102456ca8ce541439eb1948d0846e2967564c41/nxdrive/client/local/windows.py#L122) to keep the same flow (logging mostly).